foreman/data_refinery_foreman/foreman/management/commands/run_tximport.py

```diff
@@ -77,7 +74,7 @@ def run_tximport():
                 # experiment will work. Tximport is somewhat special
                 # in that it doesn't actuallhy use original files so
                 # this is just used to point to the experiment.
-                samples = experiment.samples.all()
+                experiment.samples.all()
```

Comment:
This was a dead query. Did we want it to do something?

Reply:
looks like we wanted to use it below but added the whole value instead. Will remove the whole line